### PR TITLE
Remove camelCase from Rust structs/enums generated from protobuf.

### PR DESCRIPTION
**Pull request by fmassot**
`make test-all` is passing.

### CODE REVIEW

File: quickwit-cli/tests/cli.rs [hunk 1]
```diff
@@ -797,7 +797,7 @@ async fn test_all_local_index() -> Result<()> {
     assert_eq!(result["num_hits"], Value::Number(Number::from(2i64)));
 
     let search_stream_response = reqwest::get(format!(
-        "http://127.0.0.1:{}/api/v1/{}/search/stream?query=level:info&output_format=csv&fast_field=ts",
+        "http://127.0.0.1:{}/api/v1/{}/search/stream?query=level:info&output_format=Csv&fast_field=ts",
```
Comment (fmassot) [inline quickwit-cli/tests/cli.rs:800]: This is the only thing that is a bit annoying here, we need to give `Csv` or `ClickHouseBinaryFormat` for the output format. I'm ok with that.

File: quickwit-proto/build.rs [hunk 2]
```diff
@@ -26,10 +26,7 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
     // prost_config.type_attribute("LeafSearchResponse", "#[derive(Default)]");
     prost_config.protoc_arg("--experimental_allow_proto3_optional");
     tonic_build::configure()
-        .type_attribute(
-            ".",
-            "#[derive(Serialize, Deserialize)]\n#[serde(rename_all = \"camelCase\")]",
```
Comment (fmassot) [inline quickwit-proto/build.rs:31]: remove camelCase

File: quickwit-cli/tests/cli.rs (hunk 1 above)
Comment (guilload) [reply quickwit-cli/tests/cli.rs:800]: Can't we apply `#[serde(rename_all = snake_case)]`? on the `OutputFormat` enum?

Comment (fmassot) [reply quickwit-cli/tests/cli.rs]: Yes, you're right!

**Comment by fmassot** [comment]
@guilload PTAL

**Review by guilload — APPROVED** [review]

**Pull request merged**